packages/create-payload-app/src/lib/create-project.spec.ts

```diff
@@ -155,46 +155,35 @@ describe('createProject', () => {
         expect(content).toContain(dbReplacement.configReplacement().join('\n'))
       })
     })
+
     describe('managing env files', () => {
-      it('updates .env files without overwriting existing data', async () => {
+      it('generates .env with defaults', async () => {
         const envFilePath = path.join(projectDir, '.env')
         const envExampleFilePath = path.join(projectDir, '.env.example')
 
         fse.ensureDirSync(projectDir)
-        fse.ensureFileSync(envFilePath)
-        fse.ensureFileSync(envExampleFilePath)
 
-        const initialEnvContent = `CUSTOM_VAR=custom-value\nDATABASE_URI=old-connection\n`
-        const initialEnvExampleContent = `CUSTOM_VAR=custom-value\nDATABASE_URI=old-connection\nPAYLOAD_SECRET=YOUR_SECRET_HERE\n`
-
-        fse.writeFileSync(envFilePath, initialEnvContent)
-        fse.writeFileSync(envExampleFilePath, initialEnvExampleContent)
+        if (fse.existsSync(envFilePath)) fse.removeSync(envFilePath)
```

Comment:
nitpick: would be nice to move these repetitive lines and reused `const` values into a `beforeEach` in this `describe` block.

Reply:
Done in [a0fb7af](https://github.com/payloadcms/payload/pull/12572/commits/a0fb7af7ddd4f88947da583e82a0a9ca7e8321f2).